openshift/README.md

```diff
@@ -61,6 +61,17 @@ or
 make push-prod
 ```
 
+Alternatively, when working with github repo, the image can be published
+by simply creating a tag with name respecting [semver](https://semver.org/) convention.
```

Comment:
```suggestion
by simply creating a tag with name respecting [semver](https://semver.org/) convention
and starting with 'v' prefix, i.e. `v0.0.4`.
```

Just to be super-explicit b/c for example I don't prefix my tags with 'v' and still follow semver :)